.travis.yml

```diff
@@ -28,5 +28,8 @@ before_script:
   - mysql -e 'create database nolotirov3_test;'
   - cp config/secrets.yml.example config/secrets.yml
   - cp config/database.yml.travis config/database.yml
+branches:
+  only:
+    - master
```

Comment:
No. If you look at other PR's, Travis is running twice, once for the branch and once for the PR. So the time to get the green tick is doubled.

This PR:

![captura de pantalla de 2016-03-17 12 49 00](https://cloud.githubusercontent.com/assets/2887858/13845203/af4bb93e-ec3e-11e5-888f-2b135d7fac6b.png)

Other PRs

![captura de pantalla de 2016-03-17 12 48 49](https://cloud.githubusercontent.com/assets/2887858/13845209/b948c896-ec3e-11e5-94aa-526fe9508409.png)